qiskit/aqua/operators/converters/abelian_grouper.py

```diff
@@ -88,30 +87,33 @@ def group_subops(cls, list_op: ListOp, fast: bool = True, use_nx: bool = False)
 
         Args:
             list_op: The Operator to group into Abelian groups
-            fast: Enable the fast commutation graph generation if all operators are Pauli operators
-            use_nx: Enable networkx.coloring.greedy_color instead of the numpy-based coloring
+            fast: DEPRECATED. Enable the fast commutation graph generation if all operators are
+                Pauli operators
+            use_nx: DEPRECATED. Enable networkx.coloring.greedy_color instead of the numpy-based
+                coloring
 
         Returns:
             The grouped Operator.
 
         Raises:
             AquaError: Any of list_op's sub-ops do not have a ``commutes`` method.
         """
-        if any(not hasattr(op, 'commutes') for op in list_op.oplist):
-            raise AquaError('Cannot determine Abelian groups if an Operator in list_op does not '
-                            'contain a `commutes` method'.format())
-
-        if fast and all(isinstance(op, PauliOp) for op in list_op.oplist):
-            edges = cls._commutation_graph_fast(list_op)
-        else:
-            edges = cls._commutation_graph(list_op)
+        if not fast or use_nx:
```

Comment:
Would it be better to make the signature so the two deprecated params, now ignored, are optional booleans ie
```
def group_subops(cls, list_op: ListOp, fast: Optional[bool] = None, use_nx: Optional[bool] = None)
```
and default them to None. Since we ignore them None is not a problem except that we can now give a warning should the user explicitly pass in either of these parameters independent of whether they had them set to the default of True or not. Then the test becomes
```
if fast is not None or use_nx is not None:
```
Also can we be more explicit on the deprecation message
```
            warnings.warn('Options `fast` and `use_nx` of `AbelianGrouper.group_subops` are '
                          'no longer used and are now deprecated and will be removed no '
                          'sooner than 3 months following the 0.8.0 release.')
```

Reply:
Thanks. It makes sense. I did it.